pages/en/lb4/Examples-and-tutorials.md

```diff
@@ -8,19 +8,10 @@ permalink: /doc/en/lb4/Examples-and-tutorials.html
 summary:
 ---
 
-Currently, there are two main examples:
-- "Hello World" - [loopback-next-hello-world](https://github.com/strongloop/loopback-next-hello-world).  Ultimately, this will have an accompanying tutorial to help introduce developers to LB-next.
-- [loopback-next-example](https://github.com/strongloop/loopback-next-example). This will demonstrate how to design your infrastructure using a microservices with LoopBack-next
-
-**Tutorials**
-- [Testing Your Application](Testing-your-application.html)
-
-Not yet created:
-
-- Setting up Routes
-- Application Logic
-- Persisting Data with Repositories
-- Orchestrating REST and SOAP Services
-- Defining Who Can Access What
-- Repository Composition
-- Installing Connectors
+Currently, there are two main tutorials:
```

Comment:
There are 3 listed here, not 2.  Perhaps reword to avoid future similar issues:
"The LoopBack 4 tutorials are:"